ERROR - 2023-10-19 15:34:17 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'rest-perpustaka...', 3306, '', 0)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#9 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-19 15:34:17 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
ERROR - 2023-10-19 15:34:53 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'rest-perpustaka...', 3306, '', 0)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#9 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-19 15:34:53 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
ERROR - 2023-10-19 15:34:57 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'rest-perpustaka...', 3306, '', 0)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#9 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-19 15:34:57 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
ERROR - 2023-10-19 15:35:02 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'rest-perpustaka...', 3306, '', 0)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#9 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-19 15:35:02 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
ERROR - 2023-10-19 15:36:01 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'rest-perpustaka...', 3306, '', 0)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#9 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\REST\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\REST\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\REST\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\REST\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\REST\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\REST\app\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\REST\system\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
#7 C:\xampp\htdocs\REST\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
#8 C:\xampp\htdocs\REST\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\REST\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\REST\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-10-19 15:36:01 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth\Login.php(18): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Auth\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\Login))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
CRITICAL - 2023-10-19 15:47:26 --> Call to undefined method App\Models\Peminjaman::calculateDenda
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Crud\PeminjamanController.php(94): CodeIgniter\Model->__call('calculateDenda', [...])
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Crud\PeminjamanController->update('AGT004')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud\PeminjamanController))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
CRITICAL - 2023-10-19 15:48:43 --> Call to undefined method App\Models\Peminjaman::calculateDenda
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Crud\PeminjamanController.php(94): CodeIgniter\Model->__call('calculateDenda', [...])
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Crud\PeminjamanController->update('AGT004')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud\PeminjamanController))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
CRITICAL - 2023-10-19 15:49:41 --> Call to undefined method App\Models\Peminjaman::calculateDenda
in SYSTEMPATH\Model.php on line 838.
 1 APPPATH\Controllers\Crud\PeminjamanController.php(94): CodeIgniter\Model->__call('calculateDenda', [...])
 2 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Crud\PeminjamanController->update('AGT004')
 3 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud\PeminjamanController))
 4 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\REST\\public\\index.php')
